functional_tests/identify.test.ts

```diff
@@ -24,6 +25,30 @@ test('identify sends a identify event', async () => {
             })
         )
     )
+    expect(jest.mocked(logger).error).toBeCalledTimes(0)
+})
+
+test('identify warns and succeed when given a number distinct_id', async () => {
```

Comment:
Not sure if this needs to be a functional test tbh. Feels way more like unit test behaviour?

Reply:
You're right, this is done now. FYI I wrote the new tests following the newer typescript style, rather than add to `src/__tests__/posthog-core.identify.js` and left a comment explaining